guides/release/models/customizing-serializers.md

```diff
@@ -580,15 +595,17 @@ You would define your relationship like this:
 
 ```javascript {data-filename=app/serializers/post.js}
 import DS from 'ember-data';
+const { EmbeddedRecordsMixin, JSONSerializer } = DS;
```

Comment:
The `EmbeddedRecordsMixin` will be tricky, ideally we can avoid mixins altogether. We'll have to work with the Ember Data team to figure out what the future is here.

Reply:
indeed. I chatted briefly with @runspired in #dev-ember-data to get a quick port of the syntax. Same theme there.